src/developer-testing/dto/create-admin.dto.ts

```diff
@@ -0,0 +1,19 @@
+/* istanbul ignore file */
+
+import { ApiProperty } from '@nestjs/swagger';
+import { IsArray } from 'class-validator';
+
+export class CreateCddProviders {
+  @ApiProperty({
+    description: 'The addresses to create identities for and make CDD providers',
+    type: 'string',
+    isArray: true,
+    example: ['5GwwYnwCYcJ1Rkop35y7SDHAzbxrCkNUDD4YuCUJRPPXbvyV'],
+  })
+  @IsArray()
+  readonly addresses: string[];
+
```

Comment:
nit-> We can add in amount / balance attribute to specify the initial POLYX to be transferred to each Identity. And can default it to the value used in service